rotkehlchen/tests/unit/decoders/test_velodrome.py

```diff
@@ -0,0 +1,832 @@
+import pytest
+
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.evm_event import EvmEvent, EvmProduct
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.chain.evm.constants import ZERO_ADDRESS
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.chain.optimism.modules.velodrome.constants import CPT_VELODROME
+from rotkehlchen.chain.optimism.modules.velodrome.decoder import ROUTER_V1, ROUTER_V2
+from rotkehlchen.constants.assets import A_ETH, A_OP, A_WETH_OPT
+from rotkehlchen.constants.resolver import evm_address_to_identifier
+from rotkehlchen.fval import FVal
+from rotkehlchen.tests.utils.ethereum import get_decoded_events_of_transaction
+from rotkehlchen.types import ChainID, EvmTokenKind, Location, TimestampMS, deserialize_evm_tx_hash
+
+WETH_OP_POOL_ADDRESS = string_to_evm_address('0xd25711EdfBf747efCE181442Cc1D8F5F8fc8a0D3')
+WETH_OP_GAUGE_ADDRESS = string_to_evm_address('0xCC53CD0a8EC812D46F0E2c7CC5AADd869b6F0292')
+WETH_OP_LP_TOKEN = evm_address_to_identifier(
+    address=WETH_OP_POOL_ADDRESS,
+    chain_id=ChainID.OPTIMISM,
+    token_type=EvmTokenKind.ERC20,
+)
+VELO_V2_TOKEN = evm_address_to_identifier(
```

Comment:
The Velo token is not in the global DB? It should be. @yabirgb please teach how to add it and how to add it in the next global DB upgrade.

You can add it yourself but it's good to also teach Dimitries so he knows.

And I think we need a cleanup. This is what is currently in my global DB:

![2023-09-21_13-24](https://github.com/rotki/rotki/assets/1658405/bc95067b-8f38-46c9-a506-b6c91b2f7258)

There is an asset called `VELO` which is https://www.coingecko.com/en/coins/velo, and it's not our velodrome VELO.  There is `eip155:56/erc20:0xf486ad071f3bEE968384D2E39e2D8aF0fCf6fd46|VELO|velo|||1637827741|` which is exactly the same as that velo I just mentioned. It's in binance chain.

And there is `eip155:10/erc20:0x3c8B650257cFb5f272f799F5e2b4e65093a11a05|VELO|velodrome-finance|||1654113683|` which is the v1 asset.

So we should at least:

- Remove `VELO` in favor of `eip155:56/erc20:0xf486ad071f3bEE968384D2E39e2D8aF0fCf6fd46|VELO|velo|||1637827741|`. This will required DB upgrade.
- Add The v2 Velo token and have them handled as the same thing by making an asset mapping.

Do the above make sense? Not in this PR but we need either an issue for this or some kind of task. As discussed in our workshop we can easily add tasks in a release: https://github.com/orgs/rotki/projects/11/views/1?layout=board